README.md

```diff
@@ -1,49 +1,21 @@
-# Project Name
+# Portfolio Project
 
-> Description the project.
+> A copy of a Microverse porfolio template. A mobile first responsive portfolio.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- Javascript
 
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
-## Getting Started
+👤 **Author**
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-To get a local copy up and running follow these simple example steps.
-
```

Comment:
- [ ] Please make your readme file have the `getting started` section and should be more descriptive and the content should be the steps to follow to get your project up and running on a local machine. 

For example,
```
You can add the steps to follow to clone your project,

Step 1:if you dont have git installed on your machine you cad download it from [here](https://git-scm.com/downloads).

Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine


`git clone <your project link>`

Alternatively you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`

Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
```